### DISCUSSION TITLE
.NET Runtime 8.0.0

### DISCUSSION THREAD

**Discussion by morgandawn** [Tyrrrz/DiscordChatExporter#1189] [Q&A]
The latest .NET security patch has moved us up to 8.0.1 and we're having trouble getting the ChatExporter to work since it relies on 8.0.0

edited: installed an older version of 8.0.0 and the ChatExporter launched and updated. Will there be any updates to include .Net 8.0.1?

**Comment by Tyrrrz** [comment]
It runs with whatever version is the latest installed on the system
